website_docs/getting-started.md

```diff
@@ -127,11 +127,14 @@ func main() {
 		return
 	case err := <-errCh:
 		if err != nil {
-			l.Fatal(err)
+			l.Panic(err)
 		}
 	}
 }
 ```
+Note that when we receive an error on `errCh`, we call `(*log.Log).Panic` rather
+than `Fatal`. This allows deferred functions to run before the program exits,
+which we'll need when it's time to configure a [`TracerProvider`].
```

Comment:
I do not understand this exact line. Could you please elaborate?

Reply:
The reason we need a call to `Log.panic` (or a deferred call to `os.Exit` as other reviewers have suggested) is so that the `TracerProvider` we instantiate later in the tutorial can shut down gracefully in error scenarios.

However, at this point in the tutorial, the `TracerProvider` concept has not yet been introduced, so I'm indicating that this is a concept that will become relevant later.

Perhaps the following is clearer?

> This allows deferred functions to run before the program exits, functionality that we'll need shortly.